salt/resource-manager/init.sls

```diff
@@ -1,7 +1,11 @@
 {% set resource_manager_path = pillar['resource_manager']['path'] %}
 {% set map_file = resource_manager_path + "policies/map.txt" %}
 {% set policy_file_link = resource_manager_path + pillar['resource_manager']['policy_file'] %}
-{% set execs = [ 'spark-submit', 'spark-shell', 'pyspark', 'mapred', 'hive', 'beeline' ] %} 
+{% set execs = [ 'spark-submit', 'spark-shell', 'pyspark', 'mapred', 'hive', 'beeline', 'flink', 'pyflink.sh', 'start-scala-shell.sh', 'yarn-session.sh' ] %} 
+{% set flink_version = pillar['flink']['release_version'] %}
+{% set pnda_home = pillar['pnda']['homedir'] %}
+{% set flink_real_dir = pnda_home + '/flink-' + flink_version %}
```

Comment:
flink_real_dir is not used, could you just create directly flink_bin_dir then?

Reply:
will remove the  flink_real_dir  variable.  thanks